main.py

```diff
@@ -132,6 +133,9 @@ def __init__(self, *args, obj=None, **kwargs):
             tlm_slot["channel"].setText("")
             tlm_slot["value"].setText("")
 
+        if RaspberryPi is False:
+            self.comboBoxComms.setEnabled(False)
```

Comment:
Do we also need to disable if non-raspPi as well? i.e. default it to Serial and then disable selection?

Reply:
But what happens if you select serial on the RaspPi and try to run the code? Does it crash when you send something?